src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -444,3 +459,8 @@ def load_command_table(self, _):
 
     with self.command_group('vm monitor log', log_analytics_data_plane_sdk, client_factory=cf_log_analytics_data_plane) as g:
         g.custom_command('show', 'execute_query_for_vm', transform=transform_log_analytics_query_output)
+
+    with self.command_group('vm monitor metrics', resource_type=ResourceType.MGMT_MONITOR, operation_group='metric_definitions', is_preview=True) as g:
```

Comment:
Provide a new way to write custom command here:
Consider importing command_type and custom_command_type in command group level and specify the corresponding command  utilizing custom_command_type with custom_command().
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands

Reply:
fix. Thanks